src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: false,
+    keySymbol: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentKeyup);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleDocumentKeyup);
+  }
+
+  handleDocumentKeyup = (e: KeyboardEvent) => {
+    if (e.key) {
+      this.setState({ pressedKey: true });
+      this.setState({ keySymbol: e.key });
+    }
+  };
+
+  render() {
+    const { pressedKey, keySymbol } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {pressedKey
```

Comment:
You can use your keySymbol instead.
If any key was pressed, your keySymbol will have some value, otherwise it will be empty string